all.json

```diff
@@ -15,6 +15,44 @@
 		"1inch-wallet.org",
 		"1inch.tk",
 		"1metamask.com",
+		"accountvalidation.pages.dev",
+		"azukis.art",
+		"bayc-land.mints-nft.live",
+		"bnbfaucet.site",
+		"bonowatch.com",
```

Comment:
Not sure about this one with regards to crypto?

Reply:
all those are fake mint sites when you try to connect your wallet it gets drained

Reply:
removed them

Reply:
[bonowatch.com/docu/mail0/semtex.php](https://urlscan.io/result/7ef72149-1e35-4cf9-87fb-0d2618b2c3ca/)

https://any.run/report/9f9a3d1a4209be0ce04503d1a806330acb3a400b078ffc17235baf0d5c9701b1/29e55100-85ff-4420-a31a-69c8826aed5e